### PR TITLE
Issue874 diff hor fin

### LINKED ISSUE #874: horizontal fin model do not take into account diffuse radiation
the model HorizontalFins.mo is not taking into account the effect on the diffuse radiation, hence causing a considerable mismatch

In plot
- Red is measured value
- Green is not taking into account the effect of diffuse radiation (as implemented)
- Blue is a using the same ratio for diffuse radiation as calculated for direct radiation
- Purple is the ambient temperature

The effect is more clearly seen in the weekend period (absence of ventilation and occupancy) where the effect of the afternoon sun produces those peaks in the green plot

image

### PR DESCRIPTION
**Pull request by Mathadon**
@icupeiro can you review this?

closes #874

### CODE REVIEW

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo [hunk 1]
```diff
@@ -68,24 +76,41 @@ equation
...
+next to the window.
 </p>
 <h4>Assumption and limitations</h4>
 <p>
-We assume that the fins fully cover the window at all times.
-The fin angle should be positive.
-The diffuse solar irradiation is not modified, i.e. only the direct
-solar irradiation is influenced.
+We assume that the fins fully cover the window unless the horizontal
+displacment optin is used.
```
Comment (icupeiro) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: option

Comment (icupeiro) [reply IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: displacement

Comment (icupeiro) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: displacement option

**Comment by Mathadon** [comment]
svg is still ok.

**Pull request merged**